crates/re_ui/src/lib.rs

```diff
@@ -683,3 +683,8 @@ pub fn native_window_buttons_ui(frame: &mut eframe::Frame, ui: &mut egui::Ui) {
         frame.set_minimized(true);
     }
 }
+
+/// Is this a small-screen device?
+pub fn is_mobile(ctx: &egui::Context) -> bool {
+    ctx.screen_rect().width() < 500.0
```

Comment:
For the moment the definition of `is_mobile` isn't great here and used only as "is this a narrow screen"... which completely ignores using mobile in landscape which can be very wide!
This opens up the question what the semantics of this _really_ are. It is used for "this is a narrow screen", but one would expect `is_mobile` to be also about different controls, bigger buttons etc.

Reply:
Yeah I agree, it was badly named. I removed it.

It makes sense to have different limits for different parts of the UI anyway. We need to support a range of devices, not just "mobile" and "desktop".